torch/quantization/fx/quantize.py

```diff
@@ -108,10 +108,10 @@ def graph_module_from_producer_nodes(root, producer_nodes):
     env = {}
 
     def load_arg(a):
-        return map_arg(a, lambda node: env[node.name])
+        return map_arg(a, lambda node: env[node])
     for producer_node in producer_nodes:
-        env[producer_node.name] = graph.node_copy(producer_node, load_arg)
-    graph.output(load_arg(producer_nodes[-1].name))
+        env[producer_node] = graph.node_copy(producer_node, load_arg)
+    graph.output(load_arg(producer_nodes[-1]))
```

Comment:
@jerryzh168 you shouldn't be using the name of nodes to find correspondences between copies of the nodes in different graphs. The names are not guaranteed to be the same after copying. Please check the other places in the code where names are being used in map keys

Reply:
the map is between name in old graph to name of the corresponding node in new graph. I don't think I'm assuming the name of copied node is the same as the original node. Could you elaborate on why that matters?